_layouts/pattern.html

```diff
@@ -12,24 +12,31 @@
   {% if page.images %}
     {% case page.images.size %}
       {% when 1 %}
-      <div class="{% if page.illustration_version == 1%}large-12 medium-12{% else %}large-4 large-offset-4 medium-4 medium-offset-4{% endif %} small-12 cell">
-        <img src="{{ page.images[0].url }}" />
+      <div class="one-image {% if page.illustration_version == 1%}large-12 medium-12{% else %}large-4 large-offset-4 medium-4 medium-offset-4{% endif %} small-12 cell">
+        <!-- Adding specific class for certain images that don't fit in Firefox/Edge mobile versions -->
```

Comment:
Really great to see comments like this, makes it easy to scan through and understand what’s happening